src/steps/record-success.ts

```diff
@@ -0,0 +1,57 @@
+import crypto from 'crypto';
+import fs from 'fs';
+import { version } from '../../package.json';
+import Config from '../config';
+import { FileHasher } from '../hash';
+
+export const RECORD_SUCCESS_STEP_KEY = 'record-success-';
+const RECORD_SUCCESS_STEP_LABEL = `:up:`;
+
+function getSelfCommandLine(): string {
+  return process.env?.MONOFO_SELF || `npx monofo@${version || 'latest'}`;
```

Comment:
One of the weaker parts of this PR and concept: we're back to calling ourselves to make stuff (in this case DynamoDB queries) happen.

We could probably cook up an aws-cli command to run rather than start up monofo.